src/SentryLogger.php

```diff
@@ -96,6 +96,8 @@ public function log($value, $priority = ILogger::INFO)
 
         if ($value instanceof \Exception) {
             captureException($value);
+        } else if(is_object($value)) {
```

Comment:
* This condition matches virtually *any* object, including `ActiveRow` for example. If you try to call `captureException` with that, you'll get following error:

  ```
  Argument 1 passed to Sentry\captureException() must implement interface Throwable, instance of 
  Crm\ApplicationModule\ActiveRow given
  ```

  That means, that unless you can pass in something that implements `\Throwable` this will not work.

* If you manage to solve previous issue, please use `elseif` instead of `else if` :)

Reply:
Also, looking at current implementation of Nette, the `ParseError` you're trying to log will not be logged, as the extension compilation (`beforeCompile` method) comes way after the parsing of services. So even if you manage to match the condition here, Sentry is not initialized at the time of throwing `ParseError`.